### DISCUSSION TITLE
Welcome to Genki Study Resources Discussions!

### DISCUSSION THREAD

**Discussion by SethClydesdale** [SethClydesdale/genki-study-resources#81] [Discussions]
## 👋  Welcome!
We’re now using GitHub Discussions (which is currently in beta) as a place to connect with the members of our community. We hope that you:
  * Ask questions you’re wondering about, such as the website, the material taught in Genki, or even that mysterious stranger who works on this project.
  * Share ideas to help improve Genki Study Resources.
  * Engage with other community members, help each other out, and make new friends.
  * Welcome others and have fun learning together!

To get started, comment below with an introduction and tell us about yourself. Bonus points if you can do it in Japanese!

-----

### Note

This is only the welcoming topic, you can access the full forum [here](https://github.com/SethClydesdale/genki-study-resources/discussions). Please note, however, that a [GitHub account](https://github.com/join) is required to participate in discussions.

**Comment by SethClydesdale** [comment]
I'll go first, so you can get to know the person behind this project.

I'm Seth and I'm a self-taught web developer! 

Some of my hobbies are programming, video games, anime, and learning new languages. I'm currently self-studying Japanese and working on the 3rd Edition resources for this project. I've already completed the 2nd Edition, but decided to go through and review with the 3rd Edition thanks to [David Burela's](https://github.com/DavidBurela) generosity for gifting it to me.

Now some of my favorites..

- **Favorite Programming Language:** JavaScript. I've been at it for at least 7 years now if I'm not mistaken. I originally learned it in 2013 to make plugins for Forumactif aka Forumotion. I eventually became voluntary staff -- going by the moniker "Ange Tuteur" -- and ended up providing technical support to many customers which helped further hone my knowledge of the language.
- **Favorite Video Games:** The Legend of Heroes: Trails series, Ys (pronounced イース), Nioh, Demon's Souls, Code Vein, God Eater, Persona, and basically almost any JRPG or [Falcom game](https://www.falcom.co.jp/).
- **Favorite Animes:** Shiki, Blood+, Persona 4 (the Animation), Fate/Stay Night, Code Geass, Steins; Gate, Ghost in the Shell, Accel World, Higurashi, Ghost Hunt. My favorite genres would probably be horror, mystery, and comedy, though honestly I usually end up watching anything and liking it.

I look forward to discussing and collaborating with all of you!

**Comment by Cairee** [comment]
クライズデールさん、どもありがとうございます。 I can't even imagine the amount of work you put into making the website. You ask for suggestions when I already see the website as flawless. I use it every day in both the morning and afternoon. Your work is greatly appreciated for its incredible price of $0, keep up the good work. 

はじめまして, クレアです. 日本語を１年間勉強しています。去年、日本に行きました. それは驚くべきものだった。趣味はアニメ、読書、と勉強です。

This is sadly the extent of my Japanese thus far. I wouldn't have gotten here without your website. Once again, thank you so much for putting your time into the website!

**Reply by SethClydesdale** [reply]
クレアさん、ありがとうございます！嬉しいです。Despite working on it for 2 years now, it still feels like yesterday, haha. Furthermore, it always makes my day to hear that all of that work has been of help to someone somewhere. So you're very welcome!

Your introduction is well done for a year of study. 👍 I even learned a new word from it! (驚く) I've been studying casually for about 2 years now myself and I'd say I'm only somewhere between beginner and intermediate level. There's still so much to learn, so let's both keep it up. 💪 

You're welcome once again and thanks for sharing your introduction. よろしくお願いします！

**Comment by violinfire** [comment]
こんにちは。はじめまして。わたしは ハディヤです。にじゅうさんさいです。きょうしです。よろしくおねがいします。
I started leaning Japanese last month. Haha, please make corrections if you see any. I am just starting to learn Kanji on WaniKani, so am not yet confident enough to use them! I love to learn languages, and am also self-studying Korean, Mandarin Chinese, Swahili, and Biblical Hebrew (a beginner level in all except for Korean- my first language love!). I love using Genki so far, and I feel like it will give me a strong foundation. I "dragged" my family into learning Japanese with me, and I hope to steadily improve with them! Thank you so much for this amazing resource, I really appreciate it!
https://giphy.com/gifs/mudskipperangel-thanks-thankyou-1rP0FbWsbKqRUlIStA

**Reply by SethClydesdale** [reply]
ハディヤさんのじこしょうかいはいいです！よろしくおねがいします。:wave:

It is good for a first self-introduction! One thing to note, is that you may also use a comma in place of a period for はじめまして, and since it is _your_ self-introduction, feel free to drop わたしは. 😉 A short example: 「はじめまして、セスです。」The amazing thing about Japanese is how much less we need to say compared to English to get our thoughts across, since a lot of things are implied based on context.

Kanji can be tough, but as long as you keep at it, take your time, and use them every so often, you'll be able to master them one after the other. Genki will especially help you put those kanji you're learning to use!

You're welcome! All the best to you and your family with your studies!
